dessia_common/optimization.py

```diff
@@ -26,11 +26,12 @@ def __init__(self, attribute_name: str, value: float, name: str = ''):
 class BoundedAttributeValue(dc.DessiaObject):
     _standalone_in_db = True
 
-    def __init__(self, attribute_name: str, min_value: float, max_value: float, name: str = ''):
+    def __init__(self, attribute_name: str, min_value: float, max_value: float, number: int = 2, name: str = ''):
```

Comment:
Why this number?

Reply:
It has been added to specify the number of values the user wants to generate for this BoundedAttributeValue.
It is used here:
```
def _get_instances_numbers(self):
       return [1] * len(self.constant_attributes) + [attr.number for attr in self.sampled_attributes]
```

The two other options I see is: 

1. to specify directly in the class Sampler with a list of float, of the same length of attribute 'sampled_attributes'
2. to create a new class that inherits from BoundedAttributeValue and is only used in Sampler, with this attribute 'number'